glide.yaml

```diff
@@ -25,10 +25,15 @@ import:
   version: 1.0.5
 - package: github.com/evalphobia/logrus_sentry
   version: 0.4.5
-
 - package: github.com/prometheus/client_golang
   version: faf4ec335fe01ae5a6a0eaa34a5a9333bfbd1a30
-
+- package: github.com/alecthomas/gometalinter
+  version: 46cc1ea3778b247666c2949669a3333c532fa9c6
+- package: github.com/jteeuwen/go-bindata
+  version: 6025e8de665b31fa74ab1a66f2cddd8c0abf887e 
+- package: golang.org/x/tools/cmd/goimports
+  version: 77106db15f689a60e7d4e085d967ac557b918fb2   
```

Comment:
This is not used by our code, so shouldn't be here. 

As `go get` doesn't support versioning, we have two options for `tools` target

- Use http://labix.org/gopkg.in instead of github links
- Use `dep` instead

Reply:
> Use http://labix.org/gopkg.in instead of github links

This fails to install packages. Eg: https://gopkg.in/onsi/gomega.v1

> Use dep instead

Since these packages are not imported directly in the source code. I am unable to specify `-update/-add`  for these specific packages as per the rules described https://golang.github.io/dep/docs/ensure-mechanics.html

I have added these packages in the `Gopkg.toml` with the `[[override]]` and revision. But then this defeats the purpose for having a separate target for the tools. 

@bartoszmajsak @MatousJobanek need your thoughts and suggestions.

Reply:
Or do we write some script that does this-https://stackoverflow.com/posts/32630589/revisions?

Reply:
This sounds like a bit of overdoing

Reply:
Yeah, I agree with Bartosz. In addition, the script is not complete - you would need to compile the binary and put it to the `/bin` directory.
I was hoping that the `gopkg.in` could work, but it seems that it is not applicable for all cases... But, do we really need to lock the versions for all tools? Looking at it, the most problematic one seems to be the `gometalinter`, so if we are able to lock the version at least for this tool, then I'm fine with it.

Reply:
> But, do we really need to lock the versions for all tools?

Yes, I am not sure how much of an impact an update in tools could have on the code, but I don't think it is very significant unless there is a major change in them. 

If we decide to just lock the version for the `gometalinter`, that is possible using [`gopkg.in`](https://gopkg.in/alecthomas/gometalinter.v2). 
 
@MatousJobanek @bartoszmajsak so what do you propose?

Reply:
I vote for locking version only for `gometalinter` and for keeping the rest of the tools as it is.

Reply:
Updated `gometalinter` and undone all other changes.